prowler/lib/utils/utils.py

```diff
@@ -162,3 +164,47 @@ def is_owned_by_root(file_path: str) -> bool:
             f"{file_path}: {e.__class__.__name__}[{e.__traceback__.tb_lineno}]: {e}"
         )
         return False
+
+
+def strip_ansi_codes(s: str):
```

Comment:
What is the purpose of this function?

Reply:
To remove ANSI Codes from the color formats.